Assets/Plugins/com.playeveryware.eos.asmdef

```diff
@@ -2,18 +2,15 @@
     "name": "com.playeveryware.eos",
     "rootNamespace": "",
     "references": [
-        "GUID:be1f6e9efffcfce41aeb5c2f4c27cf75",
-        "GUID:3a63500f0eef43a438c0553491f7c5da"
+        "GUID:be1f6e9efffcfce41aeb5c2f4c27cf75"
```

Comment:
Note that what this is doing is _removing_ the reference for the now-defunct (empty) assembly with the name `com.playeveryware.eos.core`.